File: Selenium/Activity2.py
from selenium import webdriver
from selenium.webdriver.firefox.service import Service as FirefoxService

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/login-form")

    print("Page Title is: " + driver.title)

    driver.find_element (By.ID, "username").send_keys("admin")
    driver.find_element(By.ID, "passwors").send_keys("password")

    driver.find_element(By.XPATH, "//button[text()='Submit']" ).click

    message = driver.find_element(By.CSS_SELECTOR, "h1.text-center")
    print("Login message: ", message.text)

File: TestNG/Activities/Activity2.java
package Activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.SkipException;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class Activity2 {
	WebDriver driver;
	
	@BeforeClass
	public void setUp() {
		
		driver=new FirefoxDriver();
		driver.get("https://training-support.net/webelements/target-practice/");
	}
	@Test
	public void firstTest() {
		
		String title=driver.getTitle();
		System.out.println("Page Title is: " + title);
		Assert.assertEquals(title, "Selenium: Target Practice");
	}
	
		
	@Test
	public void secondTest() {
		
		WebElement blackButton=driver.findElement(By.cssSelector("button.black"));
		Assert.assertTrue(blackButton.isDisplayed());
		Assert.assertEquals(blackButton.getText(), "black");
	}

	@Test(enabled = false)
	public void thirdTest() {
		String subHeading=driver.findElement(By.className("sub")).getText();
        Assert.assertTrue(subHeading.contains("Practice"));

		
	}
	@Test
	public void fourthTest() {
        throw new SkipException("Skipping test case");

		
	}
	
	
	
	@AfterClass
    public void tearDown() {
        driver.quit();

}
}

